packages/eslint-plugin-gestalt/src/index.js

```diff
@@ -1,4 +1,5 @@
 // @flow strict
+/* eslint-disable import/no-import-module-exports */
```

Comment:
We should probably fix this in the future:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-import-module-exports.md

Reply:
@dangerismycat I think it might be ok for this file since it is the source of the `main` module: https://github.com/pinterest/gestalt/blob/master/packages/eslint-plugin-gestalt/package.json#L7
> Reports the use of import declarations with CommonJS exports in any module except for the [main module](https://docs.npmjs.com/files/package.json#main).